vocabularies/UI.json

```diff
@@ -1928,6 +1936,40 @@
                 "Property"
             ],
             "@Core.Description": "The contents of this property must not be propagated to the app-to-app navigation context"
+        },
+        "CSRFTokenRequired": {
+            "$Kind": "Term",
+            "$Collection": true,
+            "$Type": "UI.CSRFTokenRequiredType",
+            "$AppliesTo": [
+                "EntityContainer",
+                "Collection",
+                "Property"
+            ],
+            "@Core.AppliesViaContainer": true,
+            "@Core.Description": "Requests for this container, collection or property must contain a token proving that the user willingly made the request",
+            "@Core.LongDescription": "In the absence of this annotation, the token is not required for GET requests, but is required for all others."
```

Comment:
```suggestion
            "@Core.LongDescription": "In the absence of this annotation, the token is not required for GET requests, but is required for all modifying requests."
```

Reply:
At the moment, it is required even for non-modifying POST $batch requests.

Reply:
"all others" is wrong, you don't need it for HEAD and OPTIONS requests